scripts/test_updates_pg14.sh

```diff
@@ -9,3 +9,5 @@ source ${SCRIPT_DIR}/test_functions.inc
 
 run_tests -v7 \
           2.5.0-pg14 2.5.1-pg14
```

Comment:
```suggestion
run_tests -v7 \
          2.5.0-pg14 2.5.1-pg14
```
v8 should be a superset of v7 and if it works starting at 2.5.0 then  2.5+ should be removed from v7 here.